<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryDialog</class>
 <widget class="QDialog" name="BoundaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>213</width>
     <height>167</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_top">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_bottom">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_left">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_right">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_top">
          <property name="text">
           <string>10000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_bottom">
          <property name="text">
           <string>-10000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_left">
          <property name="text">
           <string>-10000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_right">
          <property name="text">
           <string>10000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Exit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sure">
        <property name="text">
         <string>Sure</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
